src/lib/support/BytesToHex.h

```diff
@@ -96,6 +96,21 @@ inline CHIP_ERROR BytesToLowercaseHexString(const uint8_t * src_bytes, size_t sr
     return BytesToHex(src_bytes, src_size, dest_hex_str, dest_size_max, HexFlags::kNullTerminate);
 }
 
+/**
+ * Convert a hex string to bytes.  Supports both lowercase and uppercase (or a
+ * mix) hex characters.
```

Comment:
```suggestion
 * mix) of hexadecimal characters. Supported range is [0-9a-fA-F].
```

Reply:
Made this change with some slight tweaks.